types/types.proto

```diff
@@ -0,0 +1,127 @@
+// Generate cmd: protoc --go_out=plugins=grpc:. types.proto
+syntax = "proto3";
+
+package types;
+
+
+// Types definitions for HyperContainer
+message ContainerPort {
+    string name         = 1;
+    int32 hostPort      = 2;
+    int32 containerPort = 3;
+    string protocol     = 4;
+    string hostIP       = 5;
+}
+
+message EnvironmentVar {
+    string env    = 1;
+    string value  = 2;
+}
+
+message VolumeMount {
+    string name       = 1;
+    string mountPath  = 2;
+    bool readOnly     = 3;
```

Comment:
options a complicated, A struct is overkill.
see also https://github.com/opencontainers/runtime-spec/blob/master/specs-go/config.go#L93-L94